deployment/network-operator/values.yaml

```diff
@@ -22,8 +22,6 @@ nfd:
 psp:
   enabled: false
 
-upgradeCRDs: true
```

Comment:
do we want to roll-back this change? what should be our defaults?

Reply:
updated the template instead